### PR TITLE
Moved all strings to translation file

### PR DESCRIPTION
**Pull request by ignaciosantise**
## Summary
* Centralized all texts in one file, this will make easier to add support to new languages in the future
* Moved some constants to root folders
* Solved a warning of keys
* Some code styling
* Added rn-config file to repo

## Notion Ticket
[Link](https://www.notion.so/860acccea5a8443aa9479260b8a453fb?v=df094875b8b34fa99c81f2b23171c393&p=76b44d8a50914034844caa293d96bd5d)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +81,7 @@ The code is built using React-Native and running code locally requires a Mac or
     - You do **not** need CocoaPods since you already installed it.
 
   ```bash
-  yarn ios
+  cd ios && pod install
```
[the remark this answers is not in the document]
Comment (ignaciosantise) [reply README.md:84]: 😆  right, we could remove this part, and maybe improve the readme later

File: src/screens/tabs/Profile/screens/Settings/index.js [hunk 2]
```diff
@@ -63,32 +65,35 @@ const Settings = () => {
...
+      },
+      {
+        icon: '🔒',
+        name: t('settings.items.lock.name'),
+        description: t('settings.items.lock.desc'),
+        onPress: lockAccount,
+      },
+    ],
+    [],
+  );
```
Comment (mattgle) [inline src/screens/tabs/Profile/screens/Settings/index.js:96]: why useMemo here?

File: src/screens/tabs/components/WalletHeader/index.js [hunk 3]
```diff
@@ -42,26 +44,29 @@ const WalletHeader = () => {
...
+      },
+      {
+        image: <Icon name="swap" />,
+        colors: [Colors.Rainbow.Green, Colors.Rainbow.Teal],
+        text: t('common.swap'),
+        disabled: true,
+      },
+    ],
+    [],
+  );
```
Comment (mattgle) [inline src/screens/tabs/components/WalletHeader/index.js:69]: same here, just asking

File: src/screens/tabs/Profile/screens/Settings/index.js (hunk 2 above)
Comment (ignaciosantise) [reply src/screens/tabs/Profile/screens/Settings/index.js:96]: because in every render of the function, a new object is created, and its not necessary

**Review by mattgle — APPROVED** [review]
Great job! This is huge, just one question, but the PR is approved!

File: src/screens/tabs/Profile/screens/Settings/index.js (hunk 2 above)
Comment (mattgle) [reply src/screens/tabs/Profile/screens/Settings/index.js:96]: Perfect bro!

File: src/screens/tabs/components/WalletHeader/index.js (hunk 3 above)
Comment (mattgle) [reply src/screens/tabs/components/WalletHeader/index.js:69]: solved

**Pull request merged**